.changeset/wild-cycles-care.md

```diff
@@ -0,0 +1,5 @@
+---
+"@eth-optimism/contracts": minor
```

Comment:
> MINOR version when you add functionality in a backwards compatible manner
[Seemed about right](https://semver.org/)

Reply:
Technically we are pre major version 1 which means that semver doesn't 100% apply. We need to make sure that we track what is compatible or not, an easy way to do so would be bump the minor version each time we do a non backwards compatible regenesis until we create a major version 1 release

Reply:
So, downgrade to patch?